rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -151,32 +94,30 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
                       max_decoded_height,
                       tjpf,
                       TJFLAG_FASTDCT, &x1_diff, &crop_width_diff,
-		                  x1, y1, crop_width, crop_height) != 0)
-
-    {
+                      _crop_window.x, _crop_window.y, _crop_window.W, _crop_window.H) != 0) {
         WRN("Jpeg image decode failed " + STR(tjGetErrorStr2(m_jpegDecompressor)))
         return Status::CONTENT_DECODE_FAILED;
     }
 
-    if (x1 != x1_diff) {
-        //std::cout << "x_off changed by tjpeg decoder " << x1 << " " << x1_diff << std::endl;
+    // x1-diff should be set to x offset in tensor pipeline and removed.
```

Comment:
Add a comment that this will be set as offset in tensor pipeline and the calculation will be removed.